content/en/platform/corda/4.4/open-source/corda-network/UAT.md

```diff
@@ -55,4 +55,4 @@ Corda Network Pre-Production is governed by an [independent Foundation](https://
 
 Steps to join are outlined on the [Corda Network site](https://corda.network/participation/index.html).
```

Comment:
Apologies if I missed this in the earlier review - this link doesn't work and should be replaced with
https://corda.network/joining-corda-network/onboarding-workflow

Reply:
Changed throughout